plugins/examples/stable-analysis/src/main/java/org/elasticsearch/example/analysis/lucene/SkipTokenFilter.java

```diff
@@ -14,16 +14,18 @@
 
 import java.io.IOException;
 
-public class Skip1TokenFilter extends FilteringTokenFilter {
+public class SkipTokenFilter extends FilteringTokenFilter {
```

Comment:
Can we make this name more descriptive? Isn't this specifically skipping a certain ascii digit?